scripts/auto-detect-cluster.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+usage() {
+    echo "------------------------"
+    echo "Script to check which AKS cluster is active"
+    echo "------------------------"
+    echo "Usage: $0 -r <aksResourceGroupName> -k <aksClusterName> -e <aksEnvironmentName>"
+    echo "------------------------"
+    return 1
+}
+OPTIND=1
+
+while getopts r:k:e: o; do
+    case "$o" in
+        r)
+            aksResourceGroupName="$OPTARG"
+            ;;
+        k)
```

Comment:
could we add long arguments please and use those instead so they are self explanatory when reading the code?

Code is read many more times than it is written / edited, making it clear to the reader would be better especially given i don't see how / why `k` is even used here?

Reply:
Script updated to use `getopt` which allows for long option inputs, inputs given sensible names to make things easier for end users and steps template updated to use long options so its obvious what they are doing.
Working pipeline shown here: `https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=477602&view=results`